bom/pom.xml

```diff
@@ -120,8 +120,8 @@
         <postgressql.version>42.4.1</postgressql.version>
 
         <prometheus.version>1.10.0</prometheus.version>
-        <spring-boot.version>2.7.4</spring-boot.version>
-        <spring-security.version>5.5.5</spring-security.version>
+        <spring-boot.version>2.7.8</spring-boot.version>
+        <spring-security.version>5.5.7</spring-security.version>
```

Comment:
Unless there is a pressing reason not to, we should rely on version that spring boot proposes (5.7.6).
I am not quite sure why we should need to specify the version at all, maybe because some application will not rely on spring boot.

Reply:
changed